packages/front-end/components/DemoDataSourcePage/DemoDataSourcePage.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { FC, useCallback, useState } from "react";
+import LoadingSpinner from "@/components/LoadingSpinner";
+import DeleteButton from "@/components/DeleteButton/DeleteButton";
+import { useDefinitions } from "@/services/DefinitionsContext";
+import { useAuth } from "@/services/auth";
+import { useDemoDataSourceProject } from "@/hooks/useDemoDataSourceProject";
+
+type DemoDataSourcePageProps = {
+  error: string | null;
+  success: string | null;
+  ready: boolean;
+  exists: boolean;
+  onCreate: () => void;
+  onDelete: () => void;
+};
+
+export const DemoDataSourcePage: FC<DemoDataSourcePageProps> = ({
+  onCreate,
+  onDelete,
+  success,
+  error,
+  ready,
+  exists,
+}) => {
+  return (
+    <div className="container-fluid pagecontents">
+      <h1>Demo Datasource</h1>
+
+      <div className="card p-4">
+        {/* Intro section */}
+        <p>Create a demo datasource project or delete the existing one.</p>
+        <p>
+          There are some restrictions when creating resources in this project.
+        </p>
+        <p>
+          All created resources will be deleted when the project is deleted.
+        </p>
+        <p>
+          If you accidentally delete something in the demo project and would
```

Comment:
"accidentally delete one of our sample metrics, experiments or features and would like to restore it..."

Reply:
Change made here 667afcdbf13a32e1f5d4c329a9cc7a8c3e7d5816